test/performance/Benchmarks.md

```diff
@@ -0,0 +1,62 @@
+# Benchmarks
+
+Knative performance benchmarks are tests geared towards producing useful
+performance metrics of the knative system. All the raw metrics are stored in
+[mako](https://github.com/google/mako)
+
+## Existing scripts
+
+There are two existing scripts that help in recycling the benchmark clusters
+and have latest knative/serving updates:
+
+1. [update.sh](https://github.com/knative/serving/blob/master/test/performance/tools/update-serving): Update all the benchmark clusters every hour
+2. [recreate.sh](https://github.com/knative/serving/blob/master/test/performance/tools/recreate-serving): Re-create the benchmark cluster once a day
```

Comment:
There aren't particularly relevant to how I author a new benchmark, they probably belong elsewhere.